### DISCUSSION TITLE
how to work with "group-with-active"

### DISCUSSION THREAD

**Discussion by hatiwowo** [unocss/unocss#3289] [Q&A] [answered]
How to achieve the effect achieved in tailwind in unocss
https://play.tailwindcss.com/1zmuDZtToo

when parent node has class "active", how to make child node class "group-with-active" works?

**Comment by chu121su12** [comment]
Try with the generic [bracket syntax](https://unocss.dev/play/?html=DwEwlgbgBAxgNgQwM5ILwCIDmAnA9gVwAcoEYAXSAU3QD4AoKKUSWRFDHAwgWgG0A6UhQiUAutzKUAHmW7ZKIbgFYADCvRA&config=PTAEFMGcBsEsDsAuBaAJrSBDARtcz5wAPFOQ0eAe2QCcpEbYBjRcVZWAWwAdKbFIAKC69%2BoAN6DQoVOABmCcAGFK8BQHMANFNDc6kcIgCCiBrGwBXRLDkBPbdL1RDASSarID3fsMBVKtoAvqByNJScoADkFlRMkJCRgoLEoogy8pgW0GmyCoQqarDqABSS0jRZUABcoADaOtK1kUwWkIjhtJWRmhKg7tB8NZF0qJGggQC6XlM6kAAWfIgtAjVl0lEtbR3zi8uRQ6wkyNDqoIcofJjw6uCgCwBu4DRV58ismNDdOoFeTgYrdQa3mciH8lGKAEovI4fMZTIxLNY7JDocD-m4PKUgdJIEwPuAagBGAB0ACZUdImKh4EM5qZuJAqiAoJxifNgF91uMoToZoEIYIgA&css=PQKgBA6gTglgLgUzAYwK4Gc4HsC2YDCAyoWABYJQIA0YAhgHYAmYcUD6AZllDhWOqgAOg7nAB0YAGLcwCAB60cggDYIAXGBDAAUKDBi0mXGADe2sGC704AWgDuCGAHNScDQFYADJ4Dc5sAACtMLKAJ5gggCMLPK2ABR2pPBIcsoAlH4WAEa0yADWTlBYqEw2yFjK3Bpw5LxxAOTllVDoYpSMYgAs3vUZ2gC%2BmsBAA&options=N4IgLgTghgdgzgMwPYQLYAkyoDYgFwJTZwCmAvkA).

**Reply by hatiwowo** [reply, marked as answer]
it works! thanks!
